modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,137 @@ endif::[]
 
-In addition to Mule OAuth 2.0 Provider, MuleSoft offers out-of-the-box integration with PingFederate, OpenAM and Open ID Connect OAuth 2.0 Providers.
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
+
+* PingFederate
+* OpenAM
+* OAuth 2.0 providers supporting Open ID Connect
+
+The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] is designed to work with the Mule OAuth 2.0 provider.
+
+== Downloading the Mule OAuth 2.0 Provider
+
+You can download the Mule OAuth 2.0 provider from Anypoint Exchange:
+
+* Mule 3.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform].
+* Mule 4.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
++
+This feature is availble from Mule 4.2.0.
+
+== OAuth 2.0 dance
+
+The authentication performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This authentication process is known as the xref:oauth-dance-about.adoc[OAuth 2.0 dance]. 
+
+Mule OAuth 2.0 provider supports all xref:oauth-grant-types-about.adoc[grant types]. The entity that requests access to a resource protected by Mule OAuth 2.0 access token enforcement policy is a `Client`.
+
+=== Endpoints
+
+The Mule OAuth 2.0 provider has the following endpoints that you can call in your API implementation:
+
+image::mule-oauth-provider.png[height=150,width=250]
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Purpose | Default path | Description
+| Token validation
+| /validate
+| Used to verify the validity of the token, whether the token is expired, revoked or fake. You can define more than one validation endpoint, each of them enforcing different scopes. For more information, see <<Scopes>>.
+| Authorization
+| /authorize
+| Used to obtain a token when the client is running in a browser using a scripting language, or the client is a web server.
+| Access
+| /access_token
+| Used when the client is owned by the same entity as the Mule OAuth 2.0 provider or when the client is the same entity as the protected resource owner.
+| Token revocation
+| /revoke
+| [Optional] Used to revoke a valid token to render it invalid.
+|===
+
+=== Implementing Scopes 
+
+The OAuth 2.0 https://tools.ietf.org/html/rfc6749#page-23[scopes] enable you to further limit access to a resource protected by OAuth. You can define words, such as, `READ` and `WRITE`, or others specific to your organization, such as, `CONTRACTOR`, `PUBLIC`, `EMPLOYEES_ONLY`, and so on.
```

Comment:
```suggestion
The OAuth 2.0 https://tools.ietf.org/html/rfc6749#page-23[scopes] enable you to further limit access to a resource protected by OAuth. You can use terms such as `READ` and `WRITE`, or others specific to your organization, such as `CONTRACTOR`, `PUBLIC`, `EMPLOYEES_ONLY`, and so on.
```

Reply:
Defining something is a typical coding terminology. So kept "define words" as is.